CMakeLists.txt

```diff
@@ -21,7 +29,7 @@ find_program(CLANG_TIDY_BIN clang-tidy REQUIRED)
 find_program(CLANG_FORMAT_BIN clang-format REQUIRED)
 
 set(CMAKE_EXPORT_COMPILE_COMMANDS ON)
-set(CMAKE_C_STANDARD 17)
+set(CMAKE_C_STANDARD 11)
```

Comment:
This can be removed in favour of adding `-std=gnu17` to `bpfilter_cflags` later in the file.